.vscode/cmake-variants.yaml

```diff
@@ -191,6 +191,11 @@ CONFIG:
       buildType: MinSizeRel
       settings:
         CONFIG: cubepilot_cubeorange_test
+    cubepilot_cubeorangeplus_test:
+      short: cubepilot_cubeorangeplus
+      buildType: MinSizeRel
+      settings:
+        CONFIG: cubepilot_cubeorangeplus_test
```

Comment:
Why the test one and not the default one?

Reply:
Same logic that @dagar explained here - #17745. On top of that, I figure it saves clogging the variant list with every possible variant.

Reply:
I see, thanks. I need to test that.